### ISSUE TITLE
bug: Aria label not going from IonButton to button tag | Accessibility problem

### ISSUE THREAD

**Issue by tahagoal** [ionic-team/ionic-framework#22629]
# Bug Report

**Ionic version:**
[x] **5.x**
"@ionic/react": "^5.5.1",

**Current behavior:**
I'm using a screen reader to enhance my Ionic React accessibility application and I have a button called `more` that I want to put an `aria-label` to it.
So when I put it on `IonButton` tag it's not inherited to the `button` tag, thus the screen reader still read it `more` not the `aria-label` that I put

**Expected behavior:**
The `aria-label` should be appeared in `button` tag to be readable to the screen reader

**Related code:**
```
<IonButton aria-label="Described Aria">More</IonButton>
```

**Comment by tahagoal** [comment]
![image](https://user-images.githubusercontent.com/13594710/101290065-c4f1b200-3808-11eb-97b5-e957ee1e0766.png)

**Comment by liamdebeasi** [comment]
Thanks for the issue. This has been resolved via https://github.com/ionic-team/ionic-framework/pull/22632, and a fix will be available in an upcoming release of Ionic Framework.
